README.md

```diff
@@ -1,14 +1,13 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebookresearch/habitat-api/blob/master/LICENSE)
 [![CircleCI](https://circleci.com/gh/facebookresearch/habitat-api.svg?style=shield)](https://circleci.com/gh/facebookresearch/habitat-api)
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/facebookresearch/habitat-api/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```

Comment:
Why we decided remove this shield? Looks like some people are using `good+first+issue` label as staring point.

Reply:
A github user understands how PRs work and issues is a good place to start making contribution. We don't need to explicitly state this and this is redundant information in the README.